os/arch/arm/src/amebad/amebad_reboot_reason.c

```diff
@@ -110,4 +111,13 @@ void up_reboot_reason_clear(void)
 	up_reboot_reason_write(REBOOT_REASON_INITIALIZED);
 	backup_reg = REBOOT_REASON_INITIALIZED;
 }
+
+bool up_reboot_reason_is_written(void)
+{
+	if (BKUP_Read(BKUP_REG1) != REBOOT_REASON_INITIALIZED) {
+		return true;
```

Comment:
Let's include the header for `true` and `false`.

Reply:
It is added, `stdbool.h`.